### ISSUE TITLE
Authentication error when re-ordering STATE/PROVINCE field!

### ISSUE THREAD

**Issue by nobel0** [craigchristenson/woocommerce-2checkout-api#59]
Hello dear Mr. Craig and everybody :)

Please help me fix this nightmare,, i am using checkout editor plugin and i know when using that it breaks the plugin somehow.. But i really need this checkout editor plugin.
We lost so many customers, as they want to buy but after they click place order it gives "authentication failed" error message. and stop the order process.
Of course i am sure i have out all the key right and so on, actually without the editor plugin it works okay...

I have messaged you everywhere Mr. Craig, can you please help. Please.

Thanks

**Comment by craigchristenson** [comment]
Sure buddy, sorry if I missed your message somewhere. Sounds like your using a plugin that removes the state field from the checkout. Can you give me a link to the checkout editor plugin and show me what your doing when it breaks? If I can replicate it I may be able to help.

**Comment by nobel0** [comment]
> Sure buddy, sorry if I missed your message somewhere. Sounds like your using a plugin that removes the state field from the checkout. Can you give me a link to the checkout editor plugin and show me what your doing when it breaks? If I can replicate it I may be able to help.

Thanks for the reply sir,  very appreciated.

Umm, excuse my newbie knowledge please ^^, I am using woofunnel actually and it is a paid plugin, unfortunately. Umm, actually i just did re-ordering of the state and address fields, so maybe it is from that yes.

Is it ok to enter my WP to check the plugin codes if you want? Or what do you suggest sir, i really want to make it easy so you can investigate :) ...

Basically, as soon as we edit the checkout state field like making it half width and put it above or below it did this error, btw with PayPal plugin it placed the order BUT the state is not shown in the pulled order page.

Please, i am ok to do anything so you can investigate more.

Thanks a lot for reaching out

**Comment by craigchristenson** [comment]
Sounds like the plugin's "re-ordering" function causes the state field value to not get registered with WooCommerce. Unfortunately since the state is required for some countries when using 2Checkout, we wont be able to get around this. I am happy to contact the plugins' support email if you can provide it to me or even better since you are a paid customer [you can CC me](https://github.com/craigchristenson)` on your support tickets with them. Let me know how I can help you here man.

**Comment by nobel0** [comment]
Thank you very very much for your helps and replies sir...

Update: The checkout page is now works using woofunnels BUT BUT with the default woofunnel style, if i change it it will gives that error... So i think it is not a 2checkout plugin problem, Perhaps ^^!?

Anyway sir, can you please style the plugin more, i mean like adding the card fields placeholder, like 01234... and CVC etc.. And let it only works with numbers and limit how many digits for the credit card numbers :)

I tried to do it but man it is hard for me ^^ i was trying to copy the stripe style, but i have 0 knowledge in javascript ^^..

Thanks so much
Waiting your reply sir :)
Appreciated
